### PR TITLE
Move locales into JSON file

### PR DESCRIPTION
**Pull request by rwd**

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -62,158 +62,7 @@ module.exports = {
...
-          iso: 'sl-SI'
-        },
-        {
-          name: 'Svenska',
-          code: 'sv',
-          file: 'sv.js',
-          iso: 'sv-SE'
-        }
-      ],
+      locales: require('./plugins/i18n/locales.json'),
```
Comment (monkemedia) [inline nuxt.config.js]: would it be neater if you created a locales const at the top of the page?

Comment (rwd) [reply nuxt.config.js]: Indeed, and done.

File: nuxt.config.js [hunk 2]
```diff
@@ -47,7 +48,7 @@ module.exports = {
   /*
   ** Plugins to load before mounting the App
   */
-  plugins: ['~/plugins/vue-filters', '~/plugins/i18n'],
+  plugins: ['~/plugins/vue-filters', '~/plugins/i18n.js'],
```
Comment (lbiedinger) [inline nuxt.config.js]: Is this change needed?
Shouldn't it be kept consistent between the two plugins loaded here?

Comment (rwd) [reply nuxt.config.js]: It is needed because of the existence of a plugins/i18n directory. Without it there is a warning about ambiguity of what is being required.
